StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+Internal.swift

```diff
@@ -281,8 +281,8 @@ extension EmbeddedPaymentElement.PaymentOptionDisplayData {
             label = String.Localized.apple_pay
             paymentMethodType = "apple_pay"
             billingDetails = nil
-        case .saved(let paymentMethod, _):
-            label = paymentMethod.paymentSheetLabel
+        case .saved(let paymentMethod, let confirmParams):
+            label = paymentMethod.paymentOptionLabel(confirmParams: confirmParams)
```

Comment:
What's the difference between these two? It'd be great if we could only have one, unless they have clear distinctions

Reply:
Ahh sorry this feels hacky b/c `STPPaymentMethod` represents the Stripe API PaymentMethod object. I prefer the original proposal. 

I don't have a good idea for how to fix my discomfort with `paymentSheetLabel` and `paymentOptionLabel`, so let's ship as-is for now.